MyFridge/MyFridge/Program.cs

```diff
@@ -0,0 +1,176 @@
+            }
+           
+
+        }
+        public static void Cleaning (Refrigerator refrigerator)
+        {
+            try
+            {
+                foreach (var item in refrigerator.SortOfExpiryDate())
+                {
+                    if (item.ExpiryDate < DateTime.Now) Console.WriteLine("thrown out:");
+                    Console.WriteLine(item.Name);
+                }
+                refrigerator.Cleaning();
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message + "Try again - ");
+
+            }
+        }
+        public static void InputItem(Refrigerator refrigerator)
+        {
+            Console.WriteLine("get item: get name,kind(Food Drink),kosher(Meaty Dairy Fur ) expiryDate,space");
+            try
+            {
+                string name = Console.ReadLine() ?? "it is not null ", s = (Console.ReadLine() ?? "not null"), koshrs = Console.ReadLine() ?? "not null", expiryDates = Console.ReadLine() ?? "not null", spaces = Console.ReadLine() ?? "not null";
+                KindItem kind = (KindItem)Enum.Parse(typeof(KindItem), s);
+                Kosher kosher = (Kosher)Enum.Parse(typeof(Kosher), koshrs);
+                DateTime expiryDate = DateTime.Parse(expiryDates);
+                int space = int.Parse(spaces);
+                var inputItem = new Item(name, kind, kosher, expiryDate, space);
+                refrigerator.InsertingItem(inputItem);
+                Console.WriteLine("The food entered successfully");
+            }
+             catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message + "Try again - ");
+
+            }
+        }
+        public static void HoutItem(Refrigerator refrigerator)
+        {
+            try
+            {
+                Console.WriteLine("Give an ID number of an item to take out of the fridge");
+                int id = int.Parse(Console.ReadLine() ?? "it is not null ");
+                Console.WriteLine(refrigerator.TookItem(id));
+                Console.WriteLine("The food came out successfully");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message + "Try again - ");
+
+            }
+
+        }
+            public static void SortByAvailability(List<Refrigerator> refrigerators)
+        {
+            refrigerators.Sort((p1, p2) => p1.GetFreeSpace().CompareTo(p2.GetFreeSpace()));
+            
+        }
+        static void Main(string[] args)
+        {
+            List<Item> items = new List<Item>();
+            items.Add(new Item("milk", KindItem.Drink, Kosher.Dairy, DateTime.Now, 11));
+            items.Add(new Item("pizza", KindItem.Food, Kosher.Dairy, DateTime.Now.AddDays(11), 11));
+            items.Add(new Item("woter", KindItem.Drink, Kosher.Fur, DateTime.Now.AddDays(80), 4));
+            List<Shelf> shelf = new List<Shelf>();
+            shelf.Add(new Shelf(1, items));
```

Comment:
Adding the same items to all shelves doesn't make much sense. Each item is supposed to be unique (hence has a unique id).